5m5v-bot.js

```diff
@@ -17,30 +17,57 @@ const languages = config.users.map(user => user.language);
 const pollingIntervalMs = 40 * 1000;
 const retweetDelayMs = config.delaytime || 2 * 60 * 1000;
 const isDryRun = process.argv[2] === '--dry-run';
-
 const tweetFilter = new TweetFilter(config.exclude, languages);
-const rettiwt = new Rettiwt({ apiKey: config.users[0].api_key });
 
-console.log(isDryRun ? 'Looking for new tweets (dry run)...' : 'Looking for new tweets...');
+async function getApiKey(user) {
+  if (!user.apiKey) {
+    console.log(`Logging in as ${user.username}...`);
+
+    user.apiKey = await new Rettiwt().auth.login(user.email, user.username, user.password);
```

Comment:
Even tho the project is meant for internal usage as we're having downloads of the npm package we should probably if the first user, used for fetching is set and if not send an error message

Reply:
Good point! Fixed [here](https://github.com/veganhacktivists/5m5v-bot/pull/5/commits/4c3281a45a9f2aca1abcc85e284375ce42561975).